javascript/javascript1/week4/Homework/voiceAssignment.js

```diff
@@ -0,0 +1,141 @@
+console.log("...💥...VOICE ASSISTANT🤖βeta Version...💥...");
+let myName = [];
+let toDo = [];
+const checkName = (nameInput) => {
+  if (myName.includes(nameInput)) {
+    const index = myName.indexOf(nameInput);
+    const returnName = myName[index];
+    return returnName;
+  } else {
+    myName.pop(); //I used this to keep only latest name in the list
+    myName.push(nameInput);
+    return myName[myName.length - 1];
+  }
+};
+const getReply = (command) => {
+  if (command === undefined) {
+    console.log("ERROR! Command Not Found");
+    return;
+  }
+  if (!String(command).trim()) {
+    console.log("ERROR! Command has no text");
+    return;
+  }
+  if (!isNaN(command)) {
+    console.log("ERROR! Command can not hold only numbers");
+    return;
+  }
+  const commandLowerCased = command.toLowerCase(); // Just to check conditions
+  if (command === "What is your name?") {
```

Comment:
```suggestion
  if (commandLowerCased=== "what is your name?") {
```

You have a great idea for changing the case of the `command`. Remember to use it for every test.